src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,67 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @ManyToOne
+    @JoinColumn(name = "movie_id")
+    private Movie movie;
+    @ManyToOne
+    @JoinColumn(name = "cinema_hall_id")
```

Comment:
```suggestion
   
```
this line leads to an exception. Check It out

Reply:
![Screenshot_1](https://github.com/mate-academy/hibernate-create-movie-session-hw/assets/24695868/3ee7c0e4-b0f1-498d-be5c-39510133bbfa)
idk, everyting work for me. But now I realise that I didn't get the name quite right anyway. So thanks anyway
P.S. it started to fall after I removed this line, so I'll look into it)

Reply:
these are the very magical drops, because they stopped coming out on their own